cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/16-save-and-finish.html

```diff
@@ -14,9 +14,9 @@ <h2>Great job!</h2>
     <div class="o-form_group">
         <div class="m-form-field">
             <label class="a-label a-label__heading" for="finish_link">
-                Your link
+                <strong>Your link</strong>
```

Comment:
Does this need the extra emphasis?

Reply:
I was just starting from the look of the same text on the "Save and finish" page. :) I have no real opinion here.

Reply:
Oh, I'm looking at this and realizing I put this in entirely the wrong place. Sorry! I had meant to emphasize the same text in the main template file, not this one. 🤣 😭